src/components/OrderDataTable.js

```diff
@@ -76,6 +76,11 @@ class OrderDataTable extends React.Component {
               Header: "Item Quantity",
               accessor: "ItemQuantity",
               filterable: false
+            },
+            {
+              Header: "Pickup Status",
+              accessor: "PickupStatus",
+              filterable: false
```

Comment:
picked up should also be sortable (see all not picked up)

Reply:
default sorting will not work for this column because you are trying to sort React components instead of text. Use the `sortMethod` attribute to define a custom sorting method for this column. See here for `sortMethod` https://www.npmjs.com/package/react-table